include/avif/avif.h

```diff
@@ -411,6 +411,16 @@ AVIF_API avifBool avifCleanApertureBoxConvertCropRect(avifCleanApertureBox * cla
                                                       avifPixelFormat yuvFormat,
                                                       avifDiagnostics * diag);
 
+// ---------------------------------------------------------------------------
+
+// Returns false if tiles of tileW*tileH pixels each in an image of imageW*imageH pixels violate any standards.
```

Comment:
Nit: I suggest adding parentheses to make this sentence easier to parse. Also add the words "the" and "grid".

`// Returns false if the tiles (of tileW*tileH pixels each) in a grid image of imageW*imageH pixels violate any standards.`

Or split the comment into three sentences:

`// Returns false if the tiles in a grid image violate any standards. The tiles are of tileW*tileH pixels each. The grid image is of imageW*imageH pixels.`